frontend-beeyond/app/src/app/core/services/backend-api.service.ts

```diff
@@ -99,6 +103,7 @@ export class BackendApiService {
   }
 
   getNotifications(): Observable<Notification[]> {
+    window.localStorage.setItem('lastAccess', new Date().getTime().toString());
```

Comment:
Maybe we should add a comment here - just to clarify why we need the `lastAccess` here.